### PR TITLE
Add rcon cli

### LINKED ISSUE #25: Add rcon for bungee
Hello,

I use that image in kubernetes and i want to propose add https://github.com/BorlCand/bungeecord-rcon and https://github.com/itzg/rcon-cli for send `end` when stop the container, but before implement that i just want see the interest for that

Thanks.

### PR DESCRIPTION
**Pull request by orblazer**
Hello,
I have add rcon cli on bungeecord. That use my new plugin https://github.com/orblazer/bungee-rcon.

Note: If you have an good idea to disable plugin when rcon is disabled i am open (i have think to an way for rename the plugin jar to suffix it with `.disabled`)

This close #25.

Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,18 @@ healthy
   -e PLUGINS=https://www.example.com/plugin1.jar,https://www.example.com/plugin2.jar
   ```
 
+* **ENABLE_RCON**
+
+  Enable the rcon server (that use third plugin for work [orblazer/bungee-rcon](https://github.com/orblazer/bungee-rcon), its automatically downloaded)
```
Comment (itzg) [inline README.md:70]: Minor wording tweak:
```suggestion
  Enable the rcon server (that uses a third-party plugin to work, [orblazer/bungee-rcon](https://github.com/orblazer/bungee-rcon), which is automatically downloaded)
```

File: README.md [hunk 2]
```diff
@@ -117,6 +129,43 @@ healthy
   -p 25565:25577
   ```
 
+## Interacting with the server
+
+[RCON](http://wiki.vg/RCON) is enabled by default, so you can `exec` into the container to
+access the Minecraft server console:
```
Comment (itzg) [inline README.md:135]: copy-paste left over from minecraft server image?
```suggestion
access the Bungeecord server:
```

File: run-bungeecord.sh [hunk 3]
```diff
@@ -89,6 +91,27 @@ do
...
 
+# Download rcon plugin
+if [ "${ENABLE_RCON^^}" = "TRUE" ] && [[ ! -e $BUNGEE_HOME/plugins/${RCON_JAR_URL##*/} ]]; then
+  echo "Downloading rcon plugin"
+  if ! curl -sSL -o /tmp/${RCON_JAR_URL##*/} $RCON_JAR_URL; then
+    echo "ERROR: failed to download from $RCON_JAR_URL to /tmp/${RCON_JAR_URL##*/}"
+    exit 2
+  fi
+
+  mkdir -p $BUNGEE_HOME/plugins/bungee-rcon
```
Comment (itzg) [inline run-bungeecord.sh:102]: You could move this before the curl on L97 and then `-o` directly into there. Worst case if the curl fails, it leaves behind an empty directory.

File: run-bungeecord.sh [hunk 4]
```diff
@@ -2,7 +2,9 @@
 
 : ${TYPE:=BUNGEECORD}
 : ${MEMORY:=512m}
-BUNGEE_HOME=/server
+BUNGEE_HOME=./server
```
Comment (itzg) [inline run-bungeecord.sh:5]: Was this change needed? It is supposed to line up with

https://github.com/itzg/docker-bungeecord/blob/master/Dockerfile#L4

File: run-bungeecord.sh [hunk 5]
```diff
@@ -2,7 +2,9 @@
...
+RCON_JAR_VERSION=1.0.0
```
Comment (itzg) [inline run-bungeecord.sh:6]: A trick I have learned along the way is to do this and then users can always override with a newer version if you release before this image gets updated:
```suggestion
: ${RCON_JAR_VERSION:=1.0.0}
```

File: run-bungeecord.sh (hunk 4 above)
Comment (orblazer) [reply run-bungeecord.sh:5]: Oh, that its misstake i have change the path for my test, sorry

File: README.md (hunk 2 above)
Comment (orblazer) [reply README.md:135]: Yes and visibly i have forgot some change

File: run-bungeecord.sh [hunk 6]
```diff
@@ -94,16 +94,13 @@ fi
 # Download rcon plugin
 if [ "${ENABLE_RCON^^}" = "TRUE" ] && [[ ! -e $BUNGEE_HOME/plugins/${RCON_JAR_URL##*/} ]]; then
   echo "Downloading rcon plugin"
-  if ! curl -sSL -o /tmp/${RCON_JAR_URL##*/} $RCON_JAR_URL; then
+  mkdir -p $BUNGEE_HOME/plugins/bungee-rcon
+
+  if ! curl -sSL -o "$BUNGEE_HOME/plugins/${RCON_JAR_URL##*/}" $RCON_JAR_URL; then
```
Comment (itzg) [inline run-bungeecord.sh:99]: Should this be?
```suggestion
  if ! curl -sSL -o "$BUNGEE_HOME/plugins/bungee-rcon/${RCON_JAR_URL##*/}" $RCON_JAR_URL; then
```

Comment (orblazer) [reply run-bungeecord.sh:99]: no because `/plugins/bungee-rcon` its folder for `config.yml`, all plugin need go to `/plugins`

Comment (itzg) [reply run-bungeecord.sh:99]: Oh yeah, that makes sense
